pkg/gen/input/makefile/internal/file/makefile.go

```diff
@@ -39,6 +58,35 @@ build-darwin: $(APPLICATION)-darwin
 ## build-linux: builds a local binary for linux/amd64
 build-linux: $(APPLICATION)-linux
 
+{{- if eq .CurrentFlavour .FlavourCLI}}
+
+.PHONY: package-darwin package-linux
+## package-darwin: prepares a packaged darwin/amd64 version
+package-darwin: $(APPLICATION)-package-darwin
+## package-linux: prepares a packaged linux/amd64 version
+package-linux: $(APPLICATION)-package-linux
+
+$(APPLICATION)-package-%: $(SOURCES)
+	docker run --rm -it \
+    		-v $(shell pwd):/$(APPLICATION) \
+    		-w /$(APPLICATION) \
+    		-e GOOS=$* -e GOARCH=amd64 \
+    		golang:$(GOVERSION)-alpine go build \
+    		-ldflags "$(LDFLAGS)" \
+    		-o $(APPLICATION)-$(VERSION)-$*-amd64
+	@$(MAKE) $(APPLICATION)-archive-$*
```

Comment:
This works similar to the local binary build script, but it runs in a docker container